internal/i18n/en_translations_descriptions.go

```diff
@@ -0,0 +1,544 @@
+	TokenPoolKey        = ffm("TokenPool.key", "The signing key used to create the token pool. On input for token connectors that support on-chain deployment of new tokens (vs. only index existing ones) this determines the signing key used to create the token on-chain")
+	TokenPoolSymbol     = ffm("TokenPool.symbol", "The token symbol. If supplied on input for an existing on-chain token, this must match the on-chain information")
+	TokenPoolConnector  = ffm("TokenPool.connector", "The token connector that is responsible for the token pool. Required on input when multiple token connectors are configured")
+	TokenPoolMessage    = ffm("TokenPool.message", "The UUID of the broadcast message used to inform the network to index this pool")
+	TokenPoolState      = ffm("TokenPool.state", "The current state of the token pool")
+	TokenPoolCreated    = ffm("TokenPool.created", "The creation time of the pool")
+	TokenPoolConfig     = ffm("TokenPool.config", "Input only field, with token connector specific configuration of the pool, such as an existing Ethereum address and block number to used to index the pool. See your chosen token connector documentation for details")
+	TokenPoolInfo       = ffm("TokenPool.info", "Token connector specific information about the pool. See your chosen token connector documentation for details")
+	TokenPoolTX         = ffm("TokenPool.tx", "Reference to the FireFly transaction used to create and broadcast this pool to the network")
+
+	// TokenTransfer field descriptions
+	TokenTransferType            = ffm("TokenTransfer.type", "The type of transfer such as mint/burn/transfer")
+	TokenTransferLocalID         = ffm("TokenTransfer.localId", "The UUID of this token transfer, in the local FireFly node")
+	TokenTransferPool            = ffm("TokenTransfer.pool", "The UUID the token pool this transfer applies to")
+	TokenTransferTokenIndex      = ffm("TokenTransfer.tokenIndex", "The index of the token within the pool that this transfer applies to")
+	TokenTransferURI             = ffm("TokenTransfer.uri", "The URI of the token this transfer applies to")
+	TokenTransferConnector       = ffm("TokenTransfer.connector", "The token connector. Required on input when there are more than one token connectors configured")
+	TokenTransferNamespace       = ffm("TokenTransfer.namespace", "The namespace for the transfer, which must match the namespace of the token pool")
+	TokenTransferKey             = ffm("TokenTransfer.key", "The blockchain signing key for the transfer. On input defaults to the first signing key of the organization that operates the node")
+	TokenTransferFrom            = ffm("TokenTransfer.from", "The source account for the transfer. On input defaults to the value of 'key'")
+	TokenTransferTo              = ffm("TokenTransfer.to", "The target account for the transfer. Required on input")
```

Comment:
Not technically required - it's actually `On input defaults to the value of 'key'`

For minting in particular, this is useful so you'll mint tokens to yourself unless otherwise specified. It does mean that for transfers you must specify either "from" or "to", as they can't be the same. Not sure these additional caveats are necessary to call out though.